Swiftfin/Views/VideoPlayer/VLCPlayerViewController.swift

```diff
@@ -42,12 +42,18 @@ class VLCPlayerViewController: UIViewController {
 		currentChapterOverlayHostingController?.view.alpha ?? 0 > 0
 	}
 
+	private var panBeganBrightness = CGFloat.zero
+	private var panBeganVolumeValue = Float.zero
+	private var panBeganPoint = CGPoint.zero
+
 	private lazy var videoContentView = makeVideoContentView()
 	private lazy var mainGestureView = makeMainGestureView()
 	private var currentOverlayHostingController: UIHostingController<VLCPlayerOverlayView>?
 	private var currentChapterOverlayHostingController: UIHostingController<VLCPlayerChapterOverlayView>?
+	private var systemControlOverlayLabel = UILabel()
```

Comment:
Put the construction of this label like I have for `makeVideoContentView`, it's pretty straightforward

Reply:
Oh, that's a good way